website/docs/.vitepress/theme/theme.css

```
@@ -142,6 +114,107 @@ body::-webkit-scrollbar-thumb:hover,
+    font-weight: 650;
+  }
+
+  .VPLocalNav .menu[aria-expanded='true'] {
+    background: var(--color-primary-light);
+    color: var(--color-primary);
+  }
+
+  .VPLocalNav .menu-icon {
+    /* Official Lucide Menu glyph, applied to VitePress's fixed icon slot. */
+    --icon: url("data:image/svg+xml,%3Csvg xmlns='http://www.w3.org/2000/svg' fill='none' stroke='currentColor' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' viewBox='0 0 24 24'%3E%3Cpath d='M4 6h16M4 12h16M4 18h16'/%3E%3C/svg%3E");
+    width: 18px;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)